cmd/memparse.go

```diff
@@ -49,6 +49,30 @@ func MemParse() *cobra.Command {
 		"Specify the output file to be written to",
 	)
 
+	flags.StringVarP(
+		searchPattern,
+		"search",
+		"s",
+		"",
+		"Search for a string pattern in memory pages",
+	)
+
+	flags.StringVarP(
+		searchRegexPattern,
+		"search-regex",
+		"r",
+		"",
+		"Search for a regex pattern in memory pages",
+	)
+
+	flags.IntVarP(
+		searchContext,
+		"context",
+		"c",
+		0,
+		"Print the specified number of bytes surrounding each match",
+	)
```

Comment:
@behouba would it be possible to update [checkpointctl-memparse.adoc](https://github.com/checkpoint-restore/checkpointctl/blob/037da26353b500cb14f95ccf7b3116878ab27caf/docs/checkpointctl-memparse.adoc) with information about these options?

Reply:
Sure!

Reply:
Done!